### PR TITLE
Make the JS graph show Python 2 dropping status

### PR DESCRIPTION
**Pull request by encukou**
Also:
- Remove Group graphs entirely
- Avoid the DB in the web app
- Require networkx

And, this makes the graph CPU-hungry again.

### CODE REVIEW

**Comment by hroncok** [comment]
Not sure how do I read that graph yet, but it is indeed CPU hungry :D

**Comment by hroncok** [comment]
Shouldn't the text above the graph be updated?

**Comment by encukou** [comment]
Ah, good catch! I knew I forgot something.

File: portingdb/templates/graph.html [hunk 1]
```diff
@@ -47,31 +47,13 @@
...
-                <a href='{{ url_for("package", pkg=pkg) }}'>{{ pkg }}</a>
-                package, as
-            {% endif %}
-            tracked by <a href='{{ url_for("hello") }}'>portingdb</a>.
-            Packages with no relationship to other unported packages are not
-            shown.
-            Darker colors mean more dependencies.
+            Here is a graph showing dependency relationships for
+            packages in Fedora Rawhide that need Python 2 in some way.
+            Brighter colors roughly mean the package can be removed sooner.
```
Comment (hroncok) [inline portingdb/templates/graph.html]: OK, I guess I don't get that part. I see the same colors as in portingdb. How is their brightness showing when they can be removed?

Comment (encukou) [reply portingdb/templates/graph.html]: The ones that can't be removed easily are dark (and they're in the center).
Granted, it's not very obvious – and perhaps not even helpful. Should this just use the same colors as everywhere else?
Here's the graph with this shading turned off:

btightgraph

Comment (hroncok) [reply portingdb/templates/graph.html]: I only see it on the green ones. And only now when you showed me. Better ask another person as well.

Comment (encukou) [reply portingdb/templates/graph.html]: Well, the "hard to remove" ones *are* mostly green (and gray).

Comment (encukou) [reply portingdb/templates/graph.html]: Are we bikeshedding at this point?

Comment (hroncok) [reply portingdb/templates/graph.html]: Yes. Keep the colors.

**Pull request merged**